servers/zms/schema/updates/update-20210621.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS `zms_server`.`assertion_condition` (
+  `assertion_id` INT UNSIGNED NOT NULL,
+  `condition_id` INT NOT NULL,
+  `key` VARCHAR(64) NOT NULL,
+  `operator` VARCHAR(16) NOT NULL,
+  `value` VARCHAR(2048) NOT NULL,
+  PRIMARY KEY (`assertion_id`, `condition_id`, `key`),
+  INDEX `fk_condition_assertion_idx` (`assertion_id` ASC),
+  INDEX `key_idx` (`key` ASC),
```

Comment:
why are we creating an index for the key? what's the use case for this?

Reply:
that was for before when we wanted to manipulate individual keys, will remove it.